### PR TITLE
Layout UI

### PR DESCRIPTION
**Pull request by Paulinelali**
In this PR, I:
- [x] created a component folder
- [x] created calculator.js component
- [x] rendered the created calculator.js in the App.js
- [x] added .history to .gitigore
- [x] created calculator.css

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+If you like this project...
+Give it a start and contribute
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (danmainah) [inline README.md:200]: - [ ] @Paulinelali Kindly consider adding an MIT license to your project, it's currently missing making the MIT link to rent the error below 
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Paulinelali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  Good user interface matching the design given.
✔️ Professional written README file with all sections required.


# Required Changes ♻️

- [ ] @Paulinelali I would suggest you remove the boilerplates items created when the app was created eg logo, icon, and text file(robots.txt) both in your public and src folders. It's part of the project requirements
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (Paulinelali) [reply README.md:200]: Hi, @danmainah Please can you go the extra mile by telling me specifically boiler templates to delete? I'm kind of not really sure what to do here.

Thank you so much for your help

**Comment by Paulinelali** [comment]
Thank you, I just figured that out, now I have removed them

**Review by Kamba56 — APPROVED** [review]
Hi @Paulinelali, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] Great job creating a create react app!👏👏👏
- [x] Great job adding the calculator ui!👏👏👏
- [x!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**